scripts/idl/test_matter_idl_parser.py

```diff
@@ -359,6 +373,28 @@ def test_cluster_event_acl(self):
                     ])])
         self.assertEqual(actual, expected)
 
+    def test_fabric_sensitive_event(self):
```

Comment:
Can you add a test for a fabric-scoped command as well?

Reply:
test_cluster_commands on line 206 already contains that test - fabric scoped commands existed before.